docs/source/quick_start/install.rst

```diff
@@ -0,0 +1,254 @@
+
+Installation
+============
+
+For the first step of our journey, let's install MCDR
+
+.. note::
+
+    We assume you have some really basic knowledge of Python and pip.
+
+    If you have not, Google is your friend
+
+Requirements
+------------
+
+.. image:: https://img.shields.io/pypi/pyversions/mcdreforged.svg
+   :alt: Python version
+
+.. image:: https://img.shields.io/pypi/v/mcdreforged.svg
+   :alt: PyPI version
+
+MCDR is written and runs in Python 3. Make sure you have Python 3.8 or later with pip3 installed
+
+The detailed Python version requirements are as shown in the table below
+
+.. list-table::
+   :header-rows: 1
+
+   * - MCDR version
+     - Python requirement
+   * - < 2.10
+     - >= 3.6
+   * - >= 2.10
+     - >= 3.8
+
+Wait a second
+-------------
+
+In some "tutorials", you may learned to use MCDR by downloading a zip from Github and extract it
```

Comment:
Those "tutorials" might just be outdated tutorials, and are still a tutorials. They are just outdated